libs/ffi/ffi.c

```diff
@@ -117,6 +121,68 @@ int FFI_mitls_configure(mitls_state **state, const char *tls_version, const char
     CAMLreturnT(int,ret);
 }
 
+int FFI_mitls_configure_cert_chain_file(/* in */ mitls_state *state, const char * file)
+{
+    CAMLparam0();
+    CAMLlocal2(config, camlfile);
+    int ret = 0;
+
+    camlfile = caml_copy_string(file);
```

Comment:
(not sure if that qualifies as "accessing ocaml data"... https://github.com/ocaml/ocaml/blob/trunk/otherlibs/systhreads/threads.h#L38 doesn't say)